bench.py

```diff
@@ -66,17 +66,25 @@ def printable_output(out):
     return ("%s" % out).replace('\\n', '\n').replace('\\t', '\t')
 
 
+def get_sigature(fun: str, inputs: dict) -> str:
+    ret = []
+    for e in inputs:
+        ret.append(e["internalType"])
```

Comment:
seems like a great opportunity to use a comprehension :)

```suggestion
    ret = [e["internalType"] for e in inputs]
```

Reply:
Haha, yes. I like list comprehensions!